tests/app/main/views/test_send.py

```diff
@@ -1751,7 +1751,7 @@ def test_upload_csvfile_with_valid_phone_shows_all_numbers(
     assert '202 867 0750' not in page.text
     assert 'Only showing the first 50 rows' in page.text
 
-    mock_get_notification_count.assert_called_once_with(service_one['id'])
+    mock_get_notification_count.assert_called_with(service_id=service_one['id'])
```

Comment:
Ah, is this switch because of the multiple context processors now being called?  Or was there something else that required the change?

Reply:
This switch was due to the get_notification_count() being called twice in the send workflow in `send.py`. Would there be a better way of doing it?

Reply:
No, I think you're okay!  I just wanted to make sure I understood what was happening and my assumption was correct. :-) Thanks for explaining!